.github/workflows/ci.yml

```diff
@@ -391,3 +404,29 @@ jobs:
           name: ${{ matrix.name }}_report
           path: |
             /tmp/report/
+            
+  check_driver_schema:
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-requests: write
+    needs: paths-filter
+    if: needs.paths-filter.outputs.driver_events_changed == 'true' && needs.paths-filter.outputs.driver_schema_changed == 'false'
+    steps:
+      - name: Check driver SCHEMA_VERSION
+        uses: mshick/add-pr-comment@v2
+        with:
+          message: |
+            Please double check **driver/SCHEMA_VERSION** file. See [versioning](https://github.com/falcosecurity/libs/blob/master/driver/README.VERSION.md#schema-version-number).
+          
+  check_driver_api:
+    runs-on: ubuntu-22.04
+    permissions:
+      pull-requests: write
+    needs: paths-filter
+    if: needs.paths-filter.outputs.driver_fillers_changed == 'true' && needs.paths-filter.outputs.driver_api_changed == 'false'
+    steps:
+      - name: Check driver API_VERSION
+        uses: mshick/add-pr-comment@v2
+        with:
+          message: |
+            Please double check **driver/API_VERSION** file. See [versioning](https://github.com/falcosecurity/libs/blob/master/driver/README.VERSION.md#api-version-number).
```

Comment:
How about we move these to a separate workflow and have that run on the changes to the specific files? Seems they are mostly independent and changes to files that are not the 4 new ones defined at the top of the file have no reason to run any of these checks